controller_manager:
  ros__parameters:
    update_rate: 100
    use_sim_time: true
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    x_position_controller:
      type: position_controllers/JointGroupPositionController
    y_position_controller:
      type: position_controllers/JointGroupPositionController
    duster_velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

x_position_controller:
  ros__parameters:
    joints:
      - x_joint

y_position_controller:
  ros__parameters:
    joints:
      - y_joint

duster_velocity_controller:
  ros__parameters:
    joints:
      - duster_joint
